packages/components/src/DateTimePickers/DateTime/Manager/Manager.component.js

```diff
@@ -47,7 +47,6 @@ function ContextualManager(props) {
 		const nextState = {
 			...state,
 			...newState,
-			time,
```

Comment:
This is moved to https://github.com/Talend/ui/pull/2860/files#diff-ba183129831b5094f5d0ea4646f4b316R234